src/ray/raylet/worker_pool.cc

```diff
@@ -42,7 +42,9 @@ WorkerPool::WorkerPool(
     int maximum_startup_concurrency,
     const std::unordered_map<Language, std::vector<std::string>> &worker_commands)
     : num_workers_per_process_(num_workers_per_process),
-      maximum_startup_concurrency_(maximum_startup_concurrency) {
+      multiple_for_warning_(std::max(num_worker_processes, maximum_startup_concurrency)),
```

Comment:
I feel it might be too verbose to print a warning for every num-cpus new workers. Probably doing this for every 50/100 new workers is fine enough.

Reply:
Let's try it this way and see if it's annoying. The appropriate number probably depends (linearly?) on machine size.